src/components/UnbindORCiD.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { SynapseClient, Typography } from 'synapse-react-client'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { PROVIDERS } from 'synapse-react-client/dist/containers/Login'
+import { Button, Modal } from 'react-bootstrap'
+import { onBindToORCiD } from './ORCiDButton'
+
+
+export const unbindORCiD = async (event: React.SyntheticEvent, setIsLoading: Function, orcid: string|undefined, accessToken:string|undefined, redirectAfter?:string) => {
+    event.preventDefault()
+    if(orcid){
+        try {
+            await SynapseClient.unbindOAuthProviderToAccount(PROVIDERS.ORCID, accessToken, orcid)
+            onBindToORCiD(event, setIsLoading, redirectAfter)
+        } catch(err: any){
+            console.error(err)
+        }
+    }
+}
+export type UnbindORCiDModalProps = {
+    show: boolean,
+    setShow: Function,
+    orcid: string|undefined,
+    redirectAfter?: string
+}
+
+export const UnbindORCiDDialog = (props: UnbindORCiDModalProps) => {
```

Comment:
Small nitpick, could you make the prop name and component name align?  Either change to UnbindORCiDDialogProps, or change to UnbindORCiDModal?